tap_tiktok_ads/streams.py

```diff
@@ -319,7 +321,12 @@ def do_sync(self, stream):
             advertiser_ids = self.config['accounts']
             for advertiser_id in advertiser_ids:
                 self.params['advertiser_id'] = advertiser_id
-                self.sync_pages(stream)
+                date_batches = self.get_date_batches(stream.tap_stream_id, advertiser_id)
+                for date_batch in date_batches:
+                    filter = {'create_start_time': strftime(date_batch['start_date'], FILTER_DATE_FORMAT), 
+                              'create_end_time': strftime(date_batch['end_date'], FILTER_DATE_FORMAT)}
+                    self.params['filtering'] = json.dumps(filter)
+                    self.sync_pages(stream)
```

Comment:
Let's say we are extracting `campaigns` stream and We start date is `2022-04-21` then we will miss such records.

id	create_time			modify_time
1	2022-04-18T08:32:17	2022-04-21T08:32:17

Please re-verify the logic.

Reply:
Removed those filters. As the API does not support querying using the replication key, therefore, pseudo-incremental (aka full table) logic is implemented. Now, Tap will only emit records with a replication key greater than our bookmark